lecli/query/api.py

```diff
@@ -49,6 +49,23 @@ def handle_response(response, progress_bar):
         continue_request(response, progress_bar)
 
 
+def handle_tail(response, poll_interval, poll_iteration=1000):
+    """
+    handle tailing loop
+    """
+    for i in range(poll_iteration):
+        if response_utils.response_error(response):
+            sys.exit(1)
+        elif response.status_code == 200:
+            print_response(response)
+
+        # fetch results from the next link
+        if 'links' in response.json():
+            next_url = response.json()['links'][0]['href']
+            time.sleep(poll_interval)
+            response = fetch_results(next_url)
+
```

Comment:
Then should we print an error? It will just end silently if something weird comes back